### PR TITLE
5 rxjs

### PR DESCRIPTION
**Pull request by Ripchanskiy**
https://ripchanskiy.github.io/frontend-course-2020/homeworks/serhii.ripchanskyi_Ripchanskiy/5-rxjs/
Screenshot from 2020-11-21 02-19-55

### CODE REVIEW

File: homeworks/serhii.ripchanskyi_Ripchanskiy/5-rxjs/index.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <title>5-rxjs</title>
+  <script defer src="app.js"></script>
```
Comment (DrSmile444) [inline homeworks/serhii.ripchanskyi_Ripchanskiy/5-rxjs/index.html:6]: It's better to import the script in the body end

File: homeworks/serhii.ripchanskyi_Ripchanskiy/5-rxjs/app.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const { fromEvent } = window.rxjs;
```
Comment (DrSmile444) [inline homeworks/serhii.ripchanskyi_Ripchanskiy/5-rxjs/app.js:1]: Wrap your code into IIFE to hold your scope clear

https://developer.mozilla.org/en-US/docs/Glossary/IIFE

**Review by DrSmile444 — CHANGES_REQUESTED** [review]

File: homeworks/serhii.ripchanskyi_Ripchanskiy/5-rxjs/index.html [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+            when nothing prevents our being able to do what we like best, every pleasure is to be welcomed and every
+            pain avoided. But in certain circumstances and owing to the claims of duty or the obligations of business
+            it will frequently occur that pleasures have to be repudiated and annoyances accepted. The wise man
+            therefore always holds in these matters to this principle of selection: he rejects pleasures to secure
+            other greater pleasures, or else he endures pains to avoid worse pains.
+        </p>
+    </div>
+  </div>
+  <script defer src="app.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/rxjs/6.6.3/rxjs.umd.min.js"
```
Comment (DrSmile444) [inline homeworks/serhii.ripchanskyi_Ripchanskiy/5-rxjs/index.html:124]: You need to import third-party scripts before your own scripts. So please, change the order

Everything else is great, I can merge request after this little tweak

**Review by DrSmile444 — CHANGES_REQUESTED** [review]

File: homeworks/serhii.ripchanskyi_Ripchanskiy/5-rxjs/index.html (hunk 3 above)
Comment (Ripchanskiy) [reply homeworks/serhii.ripchanskyi_Ripchanskiy/5-rxjs/index.html:123]: Done

**Review by DrSmile444 — APPROVED** [review]

**Pull request merged**